watchtower/wtclient/client.go

```diff
@@ -1056,7 +1056,11 @@ func (c *TowerClient) handleNewTower(msg *newTowerMsg) error {
 
 	// Include all of its corresponding sessions to our set of candidates.
 	sessions, err := getClientSessions(
-		c.cfg.DB, c.cfg.SecretKeyRing, &tower.ID, nil,
+		c.cfg.DB, c.cfg.SecretKeyRing, &tower.ID,
+		func(s *wtdb.ClientSession) bool {
```

Comment:
this filter could be extracted to avoid defining it twice, should we add a unit test?

Reply:
Done.